src/components/UserInfo.js

```diff
@@ -1,19 +1,26 @@
 export default class UserInfo {
-  constructor(profileNameSelector, profileJobSelector) {
+  constructor(profileNameSelector, profileJobSelector, profileAvatar) {
     this._name = document.querySelector(profileNameSelector);
-    this._occupation = document.querySelector(profileJobSelector);
+    this._about = document.querySelector(profileJobSelector);
+    this._avatar = document.querySelector(profileAvatar);
   }
 
   getUserInfo() {
     const userData = {
       name: this._name.textContent,
-      occupation: this._occupation.textContent,
+      about: this._about.textContent,
+      avatar: this._avatar.src,
```

Comment:
Needs correction: there is no need to return the avatar here, since it's controlled by a different form (you can create a separate method getAvatar)